shelterAPI.json

```diff
@@ -103,6 +100,38 @@
           }
         }
       }
+    },
+    "/shelter/announcements": {
+      "get": {
+        "tags": [
+          "Endpoints"
+        ],
+        "summary": "Gets all announcements for shelter. Requires shelter role. Gets shelter id from auth claims.",
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "404": {
+            "description": "Client error"
```

Comment:
Wait, why is this a `client error`? If I understand this correctly, this will happen when an ID in token is not present in db, right? If so, it's not really a client-side error. They somehow got a valid token with an invalid ID, so we did something wrong. So IMO this should be `500` (or `500`-something).

Reply:
Yes I agree, it should be an Unauthorized instead of a Client error